corehq/apps/api/resources/v0_5.py

```diff
@@ -345,8 +345,8 @@ def obj_delete(self, bundle, **kwargs):
 class WebUserResource(v0_1.WebUserResource):
 
     class Meta(v0_1.WebUserResource.Meta):
-        detail_allowed_methods = ['get', 'put', 'delete']
-        list_allowed_methods = ['get', 'post']
+        detail_allowed_methods = ['get']
+        list_allowed_methods = ['get']
```

Comment:
@dannyroberts 

I was trying to understand if there is actually any difference now between v0_1 and v0_5 for `WebUserResource`. Aren't they both supporting just the listing of web users now?
1. `detail_allowed_methods` and `list_allowed_methods` are already just 'get' in the parent class i.e `UserResource`
2. in serialize you don't need to handle 'POST' anymore so that method override is not needed anymore
3. I am not sure why `get_resource_uri` is needed. Its not in v0_1 which supported 'get' as well. I guess it is needed for creating the correct URL with domain param and was a bug fix but not sure.

So I think we can remove more code here.

Reply:
Thanks for pointing these out. It looks like `always_return_data = True` below also only applies to put/post requests according to https://django-tastypie.readthedocs.io/en/latest/resources.html#always-return-data, so it can be removed too now without changing the behavior.

It seems like the only real change left here is `get_resource_uri`, which I believe adds a field to the result that includes a link to the api URL specifically for that object. Not sure how important that is, but I'm okay just leaving it as it is.

I'll remove the other stuff.